docker-compose.yml

```diff
@@ -384,6 +393,19 @@ services:
         <<: *depends_on-healthy
       web:
         <<: *depends_on-healthy
+  vroom:
```

Comment:
Shouldn't we add the `vroom` address to the `x-sentry-default` key (or `web` container) as defined here?

https://github.com/getsentry/sentry/blob/39233a25d1bee5fd4dffa754b0b618128ef3deaa/src/sentry/conf/server.py#L3201

Reply:
It should be in the config file below (or through the env variable as you suggested, yes). Otherwise it won't (shouldn't?) work

Reply:
We should be good to go after this one is addressed, right? @phacops